src/utils.c

```diff
@@ -3,11 +3,15 @@
 GdkPixbuf *
 load_pixbuf (const char *name)
 {
+  char *path;
   g_autoptr(GFile) base = NULL;
   g_autoptr(GFile) file = NULL;
   g_autoptr(GFileInputStream) in = NULL;
 
-  base = g_file_new_for_path ("../textures");
+  path = g_getenv("GNOME_HEXGL_DATADIR");
```

Comment:
Missing space between` g_getenv` and opening parenthesis.

Reply:
done